tex/gregoriotex-syllable.tex

```diff
@@ -1091,13 +1091,17 @@
       \GreNoBreak%
     \fi%
     \ifgre@eolshiftsenabled%
-      %this shift will shorten the line somewhat to allow the lyrics to extend under the custos should TeX decide to insert a line break after the syllable being ended
-      \gre@calculate@eolshift{\dimexpr(\gre@dimen@enddifference-\gre@skip@bar@lastskip)\relax }%
-      \gre@debugmsg{barspacing}{End of line shift: \the\gre@dimen@eolshift}%
-      \ifdim\gre@dimen@eolshift<\gre@skip@bar@lastskip %
-        \gre@dimen@eolshift=\gre@skip@bar@lastskip\relax %
-      \fi %
-      \kern\glueexpr(-\gre@dimen@eolshift)\relax%
+      \ifdim\wd\gre@box@syllablenotes>0pt\relax%
```

Comment:
I tried using `\ifgre@endofscore` first, but didn't get any results.  Using this test did.

Spurred by your comment, I just did some further testing (taking the sample and turing it into a gabc file instead of a snippet, change pushed to the test PR) and the reason that using `\ifgre@endofscore` doesn't work is because `\GreLastOfScore` (which sets that flag to true) is never called.  If some one could look into why the command line tool is not inserting that call, I'd appreciate it.